docs/source/releases.rst

```diff
@@ -22,9 +22,20 @@ Releases
 1.1
 @@@
 
+1.1.2
```

Comment:
We haven't actually released 1.1.1 yet. Please merge this change log entry with 1.1.1.

Reply:
Err... what constitutes a release? When it is tagged on GH?

The 1.1.1 changes are already live. https://vr-spec.readthedocs.io/en/1.1/releases.html

Reply:
Yes, it*should* be when tagged.

I realized that these are live already. Next time, we should not merge until we're ready to tag. My bad.